Antidote/CallsManager.m

```diff
@@ -421,7 +435,7 @@ - (void)switchViewControllerForCall:(OCTCall *)call
 {
     AALogVerbose(@"%@", call);
 
-    if ([self.currentCallViewController isKindOfClass:[ActiveCallViewController class]]) {
+    if ([self isCurrentViewControllerReusableForCurrentCall:call]) {
```

Comment:
When you'll remove `OCTCallStatusPaused` from objcTox it will look quite nice:
```
    if ((call.status == OCTCallStatusActive) &&
        [self.currentCallViewController isKindOfClass:[ActiveCallViewController class]]) {
    ...
```
So I think you can remove `isCurrentViewControllerReusableForCurrentCall` method after updating objcTox.